CExcercises/Ex1-nIndirections.c

```diff
@@ -16,22 +16,23 @@ El resultado será 3.*/
 int nIndirections(void* initial, unsigned int indirections);
```

Comment:
Use the keyword static for functions than only need to be used in a compilation unit (meaning a .c file).